lib/onboarding/screens/log-in.dart

```diff
@@ -0,0 +1,161 @@
+import 'dart:convert';
+
+import 'package:cboard_mobile/models/error.dart';
+import 'package:cboard_mobile/screens/home_screen.dart';
+import 'package:cboard_mobile/stylesheets/constants.dart';
+import 'package:cboard_mobile/onboarding/widgets/header.dart';
+import 'package:cboard_mobile/onboarding/widgets/username-field.dart';
+import 'package:cboard_mobile/onboarding/widgets/password-field.dart';
+import 'package:cboard_mobile/shared/app-bar.dart';
+import 'package:flutter/material.dart';
+import 'package:http/http.dart' as http;
+
+// Not really sure how entrypted things should be here!
+Future<GetUserResponse> createUserLogin(String email, String password) async {
+  final response = await http.post(
+    Uri.https('jsonplaceholder.typicode.com', 'users'),
+    headers: <String, String>{
+      'Content-Type': 'application/json; charset=UTF-8',
+    },
+    body: jsonEncode(<String, String>{
+      'email': email,
+      // Should necessarily send password here too!
```

Comment:
Yup, you can send the password there. 
![image](https://user-images.githubusercontent.com/44158788/108133598-d534d180-7082-11eb-93cb-a8c120aa2499.png)